dashboard/config/application.rb

```diff
@@ -133,5 +133,9 @@ class Application < Rails::Application
       require 'newrelic_rpm'
       require 'newrelic_ignore_downlevel_browsers'
     end
+
+    # Opt-in to new Rails 4.2 error handling in transaction callbacks.
```

Comment:
Modified the first line to `# TODO: Remove once this setting becomes default, after Rails 4.2`